mint/fakeAPIData.ts

```diff
@@ -284,12 +325,51 @@ export const prevGraph: GraphDefinition = {
     }
 };
 
+export const costGraph: GraphDefinition = {
+    metadata: {
+        x_col: "cases_averted",
+        id_col: "intervention",
+        format: "long",
+        settings: ["irs_use", "net_use"]
+    },
+    series: [
+        {
+            id: "ITN",
+            y_formula: ["({population} / {procure_people_per_net}) * {price_net_standard}  + ({price_delivery} * {population} * (({procure_buffer} + 100)/ 100))"],
+            name: "Pyrethoid ITN",
+            type: "scatter",
+            marker: {color: "blue"}
+        },
+        {
+            id: "IRS + ITN",
+            y_formula: ["({population} / {procure_people_per_net}) * {price_net_standard} + ({price_delivery} * {population} * (({procure_buffer} + 100) / 100)) + 3 * ({price_irs_per_person} * {population})"],
```

Comment:
What's the ` 3 * ({price_irs_per_person}` all about? Why 3?

Reply:
Over 3 years. This is just copied blindly from Ellie's algorithm